vizro-core/examples/scratch_dev/app.py

```diff
@@ -1185,6 +1152,117 @@ def scatter_discards_parameters(data_frame, **kwargs):
 )
 
 
+# === _trigger ===
+# ====== Page using _trigger: Filter interaction over a control ======
+
+page_filter_interaction_over_control_using_trigger = vm.Page(
+    title="Filter interaction over a control using _trigger",
+    id="Filter interaction over a control using _trigger",
+    components=[
+        vm.Container(
+            title="Source graph",
+            components=[
+                vm.Graph(
+                    id="source_graph_trigger_page",
```

Comment:
```suggestion
```

Hopefully it was obvious, but the whole point of the PR is that you no longer have to write this line 😀 Now it works great without it, as expected.